mesonbuild/mparser.py

```diff
@@ -201,10 +196,7 @@ def lex(self, filename: str) -> T.Generator[Token, None, None]:
                                    "This will become a hard error in a future Meson release.")
                             mlog.warning(mlog.code_line(msg, self.getline(line_start), col), location=BaseNode(lineno, col, filename))
                         value = match_text[2 if tid == 'fstring' else 1:-1]
-                        try:
-                            value = ESCAPE_SEQUENCE_SINGLE_RE.sub(decode_match, value)
-                        except MesonUnicodeDecodeError as err:
-                            raise MesonException(f"Failed to parse escape sequence: '{err.match}' in string:\n  {match_text}")
+                        value = ESCAPE_SEQUENCE_SINGLE_RE.sub(decode_match, value)
```

Comment:
This will raise UnicodeDecodeError instead of MesonException, right? Won't that print backtrace instead of proper error?

Reply:
See the commit message -- we can't ever raise UnicodeDecodeError anymore here, the relevant code got removed ages ago.

Reply:
ok